Sources/Views/CollectionViewViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+public class CollectionViewViewCell<View: UIView>: UICollectionViewCell {
```

Comment:
Will it make sense to override `sizeThatFits` for cell and return result of view's `sizeThatFits`?

Reply:
No, cell size will be calculated using size types in any case